librz/bin/format/elf/elf.c

```diff
@@ -11,6 +11,10 @@
 #include <rz_util.h>
 #include "elf.h"
 
+#include "rz_bin_elf_is_executable.c"
+#include "rz_bin_elf_has_relro.c"
+#include "rz_bin_elf_convert_symbol.c"
```

Comment:
Either call the files `.inc` or build as individual objects instead of including.